### PR TITLE
Update the url schema for user contributions.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: marketpulse/main/fxosprice_urls.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 urlpatterns = patterns(
     '{0}.main.views'.format(settings.PROJECT_MODULE),
     url(r'^new/$', 'edit_fxosprice', name='edit_fxosprice'),
-    url(r'^all/$', 'all_fxosprice', name='all_fxosprice'),
-    url(r'^(?P<user_pk>\d+)/contributions/$', 'all_fxosprice', name='user_fxosprice'),
+    url(r'^contributions/all/$', 'view_contributions', name='all_contributions'),
```
Comment (johngian) [inline marketpulse/main/fxosprice_urls.py]: I suggest we use `list_contributions`, `list_my_contributions`.

Comment (johngian) [inline marketpulse/main/fxosprice_urls.py]: On top of that, `list_all_contributions` might be more descriptive but feel free to choose.

**Comment by johngian** [comment]
lgtm r+wc

File: marketpulse/main/templates/activities.html [hunk 2]
```diff
@@ -23,8 +23,9 @@ <h4 class="text-center">
 
   <div class="row">
     <div class="col-md-12 text-right">
-      <a href="{{ url('main:user_fxosprice', user.pk) }}" class="view-all">View your contributions</a> &bull;
-      <a href="{{ url('main:all_fxosprice') }}" class="view-all">View all contributions</a>
+      <a href="{{ url('main:user_contributions') }}"
+         class="view-all">View your contributions</a> &bull;
```
Comment (johngian) [inline marketpulse/main/templates/activities.html:28]: Do we really need to split this line? It doesn't look much bigger than previous/next ones.

File: marketpulse/main/views.py [hunk 3]
```diff
@@ -99,11 +109,11 @@ def all_fxosprice(request, user_pk=None):
         device_pk = int(device_pk)
 
     return render(request, 'fxosprice_all.html',
-                  {'user': user, 'contributions': contributions,
-                  'devices': devices,
-                  'countries': all_countries,
-                  'country_code': country_code,
-                  'device_pk': device_pk})
+                  {'contributions': contributions,
```
Comment (johngian) [inline marketpulse/main/views.py:113]: ~~It looks like `user` key is missing.~~

Comment (johngian) [inline marketpulse/main/views.py:113]: Double checked, we don't need this in the context.

File: marketpulse/main/templates/activities.html (hunk 2 above)
Comment (akatsoulas) [inline marketpulse/main/templates/activities.html:28]: I prefer a limit of 100 chars in html files. This line limit makes it OK to see all the code in half split window, in an average setup. Speaking of other lines in the codebase and 100 limit per line, there is another one that we could break :P

**Pull request merged**